pipit/trace.py

```diff
@@ -88,6 +88,13 @@ def from_csv(filename):
 
         return Trace(None, events_dataframe)
 
+    def to_chrome(self, filename=None):
```

Comment:
should probably have some other default trace name? I'm guessing with a None, it will error otherwise